tournaments/index.js

```diff
@@ -672,6 +673,16 @@ Tournament = (function () {
 		this.runAutoDisqualify();
 		this.update();
 	};
+	Tournament.prototype.onBattleJoin = function (room, user) {
+		if (this.scouting || this.isEnded || user.latestIp === room.p1.latestIp || user.latestIp === room.p2.latestIp) return;
+		var roomid = (room && room.id ? room.id : room);
+		var users = this.generator.getUsers(true);
```

Comment:
Good question. `true` in this case means that only users who weren't eliminated appear on the list. To be frank, I hate boolean attributes like that, and that probably should have been a separate function or a named parameter.